source/templates/classic/partials/followup.ejs

```diff
@@ -26,19 +26,67 @@
                 <rect x="0" y="0" width="220" height="8" fill="white" rx="5"/>
               </mask>
               <rect mask="url(#issues-bar)" x="0" y="0" width="<%= section.issues.count ? 0 : 220 %>" height="8" fill="#d1d5da"/>
-              <rect mask="url(#issues-bar)" x="0" y="0" width="<%= (section.issues.open/section.issues.count)*220 || 0 %>" height="8" fill="#238636"/>
-              <rect mask="url(#issues-bar)" x="<%= (section.issues.open/section.issues.count)*220 || 0 %>" y="0" width="<%= (1-section.issues.open/section.issues.count)*220 || 0 %>" height="8" fill="#8957e5"/>
+              <% { const {open, drafts, closed, skipped, count, collaborators = {open:0, drafts:0, closed:0, skipped:0}} = section.issues, width = 220; let x = 0; for (const {p, fill} of [
+                {p:(open-collaborators.open)/count, fill:"#238636"},
+                {p:collaborators.open/count, fill:"#56d364"},
+                {p:(drafts-collaborators.drafts)/count, fill:"#8B949E"},
+                {p:collaborators.drafts/count, fill:"#c9d1d9"},
+                {p:(closed-collaborators.closed)/count, fill:"#8957e5"},
+                {p:collaborators.closed/count, fill:"#d2a8ff"},
+                {p:(skipped-collaborators.skipped)/count, fill:"#8B949E"},
+                {p:collaborators.skipped/count, fill:"#c9d1d9"},
+              ]) { %>
+                <rect mask="url(#issues-bar)" x="<%= x %>" y="0" width="<%= p*width || 0 %>" height="8" fill="<%= fill %>"/>
+              <% x += p*width }} %>
             </svg>
-            <div class="followup legend field horizontal fill-width">
-              <div class="field center">
+            <% if ((plugins.followup.indepth)&&(section.issues.collaborators)) { %>
+              <div class="field blue fill-width">
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M5.5 3.5a2 2 0 100 4 2 2 0 000-4zM2 5.5a3.5 3.5 0 115.898 2.549 5.507 5.507 0 013.034 4.084.75.75 0 11-1.482.235 4.001 4.001 0 00-7.9 0 .75.75 0 01-1.482-.236A5.507 5.507 0 013.102 8.05 3.49 3.49 0 012 5.5zM11 4a.75.75 0 100 1.5 1.5 1.5 0 01.666 2.844.75.75 0 00-.416.672v.352a.75.75 0 00.574.73c1.2.289 2.162 1.2 2.522 2.372a.75.75 0 101.434-.44 5.01 5.01 0 00-2.56-3.012A3 3 0 0011 4z"></path></svg>
+                From communities
```

Comment:
This is the label for issues/PRs not by maintainers right?
In that case it would make more sense to say
"From community" or "From the community".

Reply:
> This is the label for issues/PRs not by maintainers right?

Indeed

> In that case it would make more sense to say
> "From community" or "From the community".

It could, not sure which is better phrasing. Depends on whether you consider each repository community or just the GitHub community as a whole